_Week5/group1/demonstration.do

```diff
@@ -0,0 +1,19 @@
+clear
+
+// Set the working directory
+global wd "/Users/Neel Desai/Desktop/untitled folder/econ490-fall22/_Group Projects/group1/
+
+// Run the do file that contains the program
+do "$wd/one_way_anova.do"
+
+// Demonstration 1
+sysuse airfare.dta, clear
+one_way_anova airfare.dta year y98
```

Comment:
Maybe this is specific to my STATA version but this dataset was not built-in for me. I had to download from [here](http://qcpages.qc.cuny.edu/~rvesselinov/statafiles.html) to run the program. It might be worthwhile to mention this in the readme file or as a comment in this do-file